### ISSUE TITLE
cell annotation

### ISSUE THREAD

**Issue by zerodohero** [vqdang/hover_net#54]
Hello, your work inspires me a lot.I'd like to ask you a quick question about cell annotation.
I now want to annotate my collection of cells, including the Outlines and types of cells.Your generosity in sharing data in your program is admirable.I want to annotate HEs cells the same way you do. Can you provide software and experience for manually annotating cells?Your help will be greatly appreciated.I wish you all the best.

**Comment by simongraham** [comment]
Hi @zerodohero 

Thanks a lot for your kind words. We used the software GIMP to do the annotation and used the pencil tool to ensure there was no aliasing. For this, we opened the original image and annotated the nuclei with a transparency layer. We didn’t use a specific schema for choosing the colours for annotating, as long as touching nuclei were not annotated with the same colour. Then we coloured the non nuclei black and exported the transparency layer. Once this is done, you can use a script to create the nuclear instance map. 

For generating the class map, we also used GIMP and used different colours for each class. Here, the nuclei outline does not need to be annotated as this has already done, but you can simply provide a dot with the appropriate colour.

**Comment by zerodohero** [comment]
Thank you very much. That's very generous of you
